src/server/hll_family.cc

```diff
@@ -71,14 +71,14 @@ OpResult<int> AddToHll(const OpArgs& op_args, string_view key, CmdArgList values
   string hll;
 
   try {
-    auto [it, inserted] = db_slice.AddOrFind(op_args.db_cntx, key);
-    if (inserted) {
+    auto res = db_slice.AddOrFind(op_args.db_cntx, key);
+    if (res.is_new) {
       hll.resize(getDenseHllSize());
       createDenseHll(StringToHllPtr(hll));
-    } else if (it->second.ObjType() != OBJ_STRING) {
+    } else if (res.it->second.ObjType() != OBJ_STRING) {
```

Comment:
do we have this check in multiple places? can we add it to inside AddOrFind as you did we FindMutable

Reply:
It doesn't seem like too many places, don't you think?

Reply:
I think that all the places that use AddOrFind should have this check, so I think it better to move it inside AddOrFind

Reply:
Ok, I'll do it in another PR though. This PR is delicate enough as is :rofl: